### ISSUE TITLE
How to test

### ISSUE THREAD

**Issue by hjdihs** [XPixelGroup/ClassSR#59]
Should we test individual branches first and then test ClassSR?When I directly test ClassSR with a pre-trained model, the test results for all branches only have green masks?How to achieve the effects mentioned in the paper
